lib/Tab.js

```diff
@@ -121,9 +126,27 @@ export default class Tab extends Component {
     )
   }
 
+  _renderBadge = () => {
+    const { active, icon, activeIcon, badgeText, badgeSize, badgeStyle, isBadgeVisible } = this.props;
+
+    if (badgeText === undefined && !isBadgeVisible) {
+      return active && activeIcon ? activeIcon : icon;
+    }
+
+    return (
+      <Badge
+        text={badgeText}
+        size={badgeSize}
+        style={badgeStyle}
+        isVisible={isBadgeVisible}
+      >
+        {active && activeIcon ? activeIcon : icon}
```

Comment:
_For android compatibility:_ The icon is rendered in Tab.js, so remove this line.

Reply:
done